parquet/src/main/java/org/apache/iceberg/parquet/ParquetValueWriter.java

```diff
@@ -28,4 +30,7 @@
   List<TripleWriter<?>> columns();
 
   void setColumnStore(ColumnWriteStore columnStore);
+
+  Stream<FieldMetrics> metrics();
```

Comment:
Documentation needed. And is the name a bit too generic?

Reply:
I'll add documentations. For the name, I think `ParquetValueWriter` already implies that it's field specific, so I guess `metrics` itself would convey the idea that it's field specific metrics. Do you have better suggestions?